app/views/tag/_tagging.html.erb

```diff
@@ -3,12 +3,21 @@
 <p>
   <span id="tags">
     <% @node.community_tags.each do |tag| %>
-      <span id="tag_<%= tag.tid %>" class="label label-primary" data-toggle="tooltip" data-html="true" title="created by <strong><%= tag.try(:author).try(:username) %></strong> <%= time_ago_in_words(Time.at(tag.date)) %> ago">
-        <a class="tag-name" href="<%= "/maps" if @node.type == "map" %>/tag/<%= tag.name %>"><%= tag.name %></a>
-        <% if current_user && (tag.uid == @node.uid || current_user.role == "admin" || current_user.role == "moderator") %>
-          <a data-confirm="This is a power tag (see https://publiclab.org/wiki/power-tags) -- and may drive a specific function on this page. Are you sure you want to delete it?" class="tag-delete" data-remote="true" href="/tag/delete/<%= @node.id %>/<%= tag.tid %>" data-tag-id="<%= tag.tid %>">x</a>
-        <% end %>
-      </span>
+      <% if tag.name.include? ':' %>
+        <span id="tag_<%= tag.tid %>" class="label label-default" data-toggle="tooltip" data-html="true" title="created by <strong><%= tag.try(:author).try(:username) %></strong> <%= time_ago_in_words(Time.at(tag.date)) %> ago">
```

Comment:
@gaeun You don't have to put a `if-else` block for the entire block just to use a different class. Just putting a conditional operator in the class itself would do. Something like
```ruby
<span class="label <%= tag.name.include?(':') ? 'label-default' : 'label-primary' %>">
```